deploy

```diff
@@ -3,684 +3,618 @@
-// Get list of all blocks
+// List Event Handlers
 // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-function list_blocks(key_id) {
+function listEventHandlers(keyId, blockId) {
     return http({
-        method  : 'GET'
-    ,   json    : true
-    ,   uri     : 'https://admin.pubnub.com/api/v1' + 
-                  `/blocks/key/${key_id}/`   +
-                  `block`
-    ,   headers : { 'X-Session-Token' : config.login.token }
-    }).catch( error => {
-        console.error( 'error', error );
-    } );
+        method: 'GET',
+        json: true,
+        uri: 'https://admin.pubnub.com/api/v1' +
+            `/blocks/key/${keyId}` +
+            `/block/${blockId}`,
+        headers: {
+            'X-Session-Token': config.token
+        }
+    }).catch(error => {
+        console.error('Failed to list event handlers'.black.bgRed)
+        console.error(error.black.bgRed);
+    });
 }
 
 // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-// Get list of all apps
+// Create Event Handler
 // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-function list_apps(account_id) {
+function createEventHandler(keyId, blockId, name, event) {
     return http({
-        method  : 'GET'
-    ,   json    : true
-    ,   uri     : `https://admin.pubnub.com/api/apps?owner_id=${account_id}`
-    ,   headers : { 'X-Session-Token' : config.login.token}
-    }).catch( error => {
-        console.error( 'error', error );
-    } );
+        method: 'POST',
+        json: true,
+        uri: 'https://admin.pubnub.com/api/v1' +
+            `/blocks/key/${keyId}/` +
+            `event_handler`,
+        headers: {
+            'X-Session-Token': config.token
+        },
+        body: {
+            key_id: keyId,
+            block_id: blockId,
+            type: 'js',
+            event: event,
+            channels: 'opengrowth.signals',
+            name: name,
+            code: "console.log('hi');",
+            output: 'output-0.5823105682419438'
+        }
+    }).catch(error => {
+        console.error('Failed to create event handler'.black.bgRed);
+        console.error(error.error.message.black.bgRed);
+    });
 }
 
 // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-// Upload Signals Handler Code
+// Get BLOCK id
 // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-function list_event_handlers(key_id, block_id) {
-    return http({
-        method  : 'GET'
-    ,   json    : true
-    ,   uri     : 'https://admin.pubnub.com/api/v1' + 
-                  `/blocks/key/${key_id}`   +
-                  `/block/${block_id}`
-    ,   headers : { 'X-Session-Token' : config.login.token }
-    }).catch( error => {
-        console.error( 'error', error );
-    } );
+async function getBlockId() {
+    let list = await listBlocks(config.keyId);
+
+    for (let i in list.payload) {
```

Comment:
`for (let block of list.payload)` would make references look like
`block.name`
instead of
`list.payload[i].name`